decidim-generators/lib/decidim/generators/install_generator.rb

```diff
@@ -158,7 +158,6 @@ def profiling_gems
 
           group :development do
             # Profiling gems
```

Comment:
Necessary change, as in the test app we do not have a Gemfile, so this bit will fail ( at least on local test_app) if we add the --profiling option

Reply:
@alecslupu Please see these comments.

Reply:
Fixed by : [00279ba](https://github.com/decidim/decidim/pull/11416/commits/00279ba04a3608e05a0f5f203540681e93be9cc8)